lib/errors/rescue_development_errors.rb

```diff
@@ -25,14 +25,10 @@ def self.rescue_from_rev_manifest(base)
       end
     end
 
-    NO_CAMPAIGNS_EXPLANATION =
-      'Error: There are no campaigns.' \
-      "\n\n" \
-      'Go to "/campaigns" to create one.'
     def self.rescue_from_no_campaigns(base)
       base.rescue_from CoursesPresenter::NoCampaignError do
-        render plain: NO_CAMPAIGNS_EXPLANATION,
-               status: 500
+        Campaign.create(title: 'Default Campaign', slug: ENV['default_campaign'])
+        redirect_to '/campaigns'
```

Comment:
Probably redirect to `/` would make the most sense, as that's generally the route where this error is encountered.

Reply:
Alright. Just a sec.

Reply:
You might also need to update `spec/lib/errors/rescue_development_errors_spec`.